### PR TITLE
switch bootnode from aws to ovh

### PR DESCRIPTION
**Pull request by v9n**
Switch bootnode on AWS to OVH nodes. Relevant infra changes are made separately.

### CODE REVIEW

File: node/res/turing-staging.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "chainType": "Live",
     "bootNodes": [
         "/ip4/54.89.28.91/tcp/30333/ws/p2p/12D3KooWKckYYKn2iztcKQcKjvdnBxs1yaL63WQigRtqMgNnmAK7",
-        "/ip4/44.205.189.85/tcp/30333/ws/p2p/12D3KooWBWgvVtvKEvHDnhvtftUVFkBZhx23PHCqcXcoqGB5f6h7"
+        "/ip4/15.204.212.186/tcp/30333/p2p/12D3KooWJSYJE71yzzw51TZb5tHb57HhbhQCxqeRRj9xU4QU1d7J"
```
Comment (chrisli30) [inline node/res/turing-staging.json:7]: so Staging currently has 1 bootnode on AWS and the other on OVH, right?

**Review by chrisli30 — APPROVED** [review]

**Review by chrisli30 — COMMENTED** [review]
Defer to @imstar15 on whether we should update genesis config.

**Review by imstar15 — APPROVED** [review]
Looks good to me!

File: node/res/turing-staging.json (hunk 1 above)
Comment (chrisli30) [reply node/res/turing-staging.json:7]: Yes, let’s run staging with two OVH servers only.

**Pull request merged**